addie/utilities/__init__.py

```diff
@@ -2,16 +2,11 @@
 from qtpy.uic import loadUi
 
 def load_ui(ui_filename, baseinstance):
+    cwd = os.getcwd()
```

Comment:
`os.getcwd()` is the directory that the user started the program from. It is all but guaranteed to not be where you expect. The `__file__` variable (set by python) is the absolute location of where the source file is.

Reply:
Yeah, quickly realized this is changing when we switch the `cwd` in the Post-Processing Tab. 

Already implementing using the `__file__` for this file. Not sure if that is the best way or if it should be something set in `scripts/addie` main script that I import here.

Testing now.